StormLib/src/SFileAddFile.cpp

```diff
@@ -100,7 +100,7 @@ static int FillWritableHandle(
     pFileEntry->dwFlags  = dwFlags | MPQ_FILE_EXISTS;
 
     // Initialize the file time, CRC32 and MD5
-    assert(sizeof(hf->hctx) >= sizeof(hash_state));
+    //assert(sizeof(hf->hctx) >= sizeof(hash_state));
```

Comment:
I don't fully understand the changes done to StormLib - but now `[HASH_STATE_SIZE]` is hardcoded, it doesn't seem to be correct for macos(arm64)? -- commenting this out doesn't break anything for either platform.

Reply:
@NEstelami perhaps you have some insights here? I believe you were the one that made the changes to StormLib, right?

Reply:
For some reason I didn't get a notification for this. There were only two real changes to StormLib - an optimization to make generating the OTRs several seconds faster, and additional checks when files are looked up in order to make `\` and `/` interchangeable. I don't think I ever made any changes to the hash state size or anything like that.